describe.go

```diff
@@ -0,0 +1,72 @@
+package pinblock
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/moov-io/pinblock/formats"
+)
+
+const (
+	DescribeEncode = "encode"
+	DescribeDecode = "decode"
+)
+
+func Describe(iso formats.ISO, w io.Writer, method, pin, account string) error {
+
+	if method == DescribeEncode {
+		return describeEncode(iso, w, pin, account)
+	} else if method == DescribeDecode {
+		return describeDecode(iso, w, pin, account)
+	}
+
+	return fmt.Errorf("the method(%s) doesn't support yet", method)
+}
+
+func describeEncode(iso formats.ISO, w io.Writer, pin, account string) error {
+
+	result, err := iso.Encode(pin, account)
+	if err != nil {
+		return fmt.Errorf("pin block encode operation failed")
+	}
+
+	tw := tabwriter.NewWriter(w, 0, 0, 2, ' ', 0)
+
+	fmt.Fprintf(tw, "PIN block encode operation finished\n")
+	fmt.Fprintf(tw, "%s\n", strings.Repeat("*", 36))
+	fmt.Fprintf(tw, "PAN\t: %s\n", account)
+	fmt.Fprintf(tw, "PIN\t: %s\n", pin)
+	fmt.Fprintf(tw, "PAD\t: %s\n", iso.Padding(pin))
+	fmt.Fprintf(tw, "Format\t: %s\n", iso.Format())
+	fmt.Fprintf(tw, "%s\n", strings.Repeat("-", 36))
+	fmt.Fprintf(tw, "Clear PIN block\t: %s\n\n", result)
```

Comment:
What is clear pin block? is it unencrypted value? If so, then for iso4 it will be encrypted.

What's the purpose of the `Describe`? Do help to debug calculation? If so, then we should find a way to extract into public methods how we calculate pin field and pan field.

As now, the only data that is not the input or output of the Encode/Decode are:
* Format()
* Padding()

are we really interested in only those two data points?

Reply:
https://github.com/moov-io/pinblock/issues/2

clear pin block means encrypted block
main purpose is to provide human-readable format of the encoding inputs and outputs.

ISO interface added to support general interface, format and padding functions are utility functions of the interface

Reply:
"Cleartext" refers to decrypted data, so "Clear" meaning encrypted is confusing to me.